DataFormats/PatCandidates/interface/MET.h

```diff
@@ -200,11 +247,25 @@ namespace pat {
       // ---- holder for pfMET specific info ---
       std::vector<SpecificPFMETData> pfMET_;
 
+      // MET significance
+      double metSig_;
+      
+      const PackedMETUncertainty findMETTotalShift(MET::METCorrectionLevel cor, MET::METUncertainty shift) const;
+   
+      std::map<MET::METCorrectionLevel, std::vector<MET::METCorrectionType> > corMap_;
```

Comment:
This is a transient map, not stored in files, only used when the combination of corrections is done. Should I still proceed?